text/_util.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { levenshteinDistance } from "./levenshtein_distance.ts";
+import { assert } from "../assert/mod.ts";
```

Comment:
@kt3k confused by this one, why is this `mod.ts` and not `assert.ts`?

Reply:
Not a strong preference, but it was more generic; I could do a find-and-replace-all from the old location to the new location, but only if I used "mod.ts" (so that imports like `assertEquals, assertThrows` kept working)

Reply:
My hope would be that in the std we try to minimize the module graph, so not sure I support this move.

Reply:
okay I'll change it